packages/helix/build.sh

```diff
@@ -30,4 +30,8 @@ termux_step_make_install() {
 
 	cp -r ./runtime "${datadir}"
 	find "${datadir}"/runtime/grammars -type f -name "*.so" -exec chmod 0700 {} \;
+
+	install -Dm 0644 "contrib/completion/hx.zsh" "${TERMUX_PREFIX}/share/zsh/site-functions/_hx"
+ 	install -Dm 0644 "contrib/completion/hx.bash" "${TERMUX_PREFIX}/share/bash-completion/completions/hx.bash"
```

Comment:
Should not the destination file be just `hx` (without the extension) ?

Reply:
Oops, my mistake, I copied it from aur's pkgbuild

Reply:
This one has the correct file name https://gitlab.archlinux.org/archlinux/packaging/packages/helix/-/blob/main/PKGBUILD?ref_type=heads